.envrc

```diff
@@ -1 +1,26 @@
-use flake
+# Check if nvm is installed, install if not
```

Comment:
fwiw. This will make anyone using nixos unable to use the envrc (unless they preinstall nvm it elsewhere). It can't just curl and install (easily). So that'll impact @nikolay-ngrok @euank @jrobsonchase @TheConcierge if that matters.

Reply:
how often do they work w/ the docs?
is there a way to have multiple `.envrc`? something like

```
direnv allow .envrc.nix
```